.travis.yml

```diff
@@ -1,11 +1,26 @@
+sudo: false
 language: node_js
-branches:
-  only:
-    - master
-os:
-    - linux
-    - osx
 node_js:
-   - "8"
-   - "6"
-   - "4"
+  - '9'
+  - '8'
+  - '7'
+  - '6'
+  - '5'
+  - '4'
+  - '0.12'
+  - '0.10'
+os:
+  - linux
+  - osx
+cache:
+  directories:
+  - node_modules
+install:
+  - node --version
+  - npm --version
+  - npm install
+script:
+  - node --version
+  - npm --version
+  - npm test
```

Comment:
You could get rid of `install` and `script` as travis will already do all this by default

Reply:
Is just to make `.travis.yml` look exacly like `appveyor.yml`.
I call it consistency. But yeah is reduntant.